docs/design/node-pool/package-based-lcm-node-pools.md

```diff
@@ -0,0 +1,23 @@
+# Package Based LCM Node Pools
```

Comment:
suggestion: Node Pools configuration for ClusterClass-based clusters

Reply:
done